cfme/tests/infrastructure/test_quota_tagging.py

```diff
@@ -318,7 +317,6 @@ def test_quota_vm_reconfigure(
         )
     provision_request = appliance.collections.requests.instantiate(request_description)
     provision_request.wait_for_request(method="ui")
-    assert small_vm.configuration != original_config
```

Comment:
**Suggestion:** 
Instead of assertion removal, you can use like this:` assert small_vm.configuration == original_config`.

As per your test, after vm reconfigure, quota gets exceeds then you can check with the original configuration of `small_vm` suggested above

Reply:
:+1: